WEB_SERVICE/WEB/_HIGH/SSL_RACCOON_ATTACK/meta.json

```diff
@@ -0,0 +1,48 @@
+{
+  "title": "Raccoon Attack on SSL/TLS",
+  "short_description": "A timing vulnerability in the Diffie-Hellman key exchange that allows attackers to recover sensitive data through precise measurement of server response times.",
+  "risk_rating": "high",
+  "references": {
+    "Raccoon Attack Research Paper": "https://raccoon-attack.com"
+  },
+  "privacy_issue": false,
+  "security_issue": true,
+  "categories": {
+    "SOC2_CONTROLS": [
+      "CC_6_1",
+      "CC_6_7",
+      "CC_7_1",
+      "CC_7_2",
+      "CC_7_3"
```

Comment:
CC_7_3 isn't related to attacks

Reply:
Sure ? This control emphasizes the importance of evaluating security events to identify potential security incidents, which includes vulnerabilities like the Raccoon Attack.